save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -294,14 +298,13 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
             }
         },
         getAdditionalDependencies = {
-            arrayOf(it.status, it.testSuite)
+            arrayOf(it.filters.status, it.filters.fileName, it.filters.testSuite, it.filters.tag)
```

Comment:
After you make `val`s instead of `var`s, you will be able to pass only `it.filters`

Reply:
Done